### PR TITLE
STDTC-14 Refactor to MirageJS

### PR DESCRIPTION
**Pull request by minkimcello**
## Motivation
Screen Shot 2020-07-14 at 4 57 15 PM
This is part of an on-going initiative to completely migrate all of `folio-org` projects from `@bigtest/mirage` to `miragejs`.

[`@bigtest/mirage`](https://github.com/bigtestjs/mirage/) was originally a prototype to make `mirage` available to any framework, but it has since been archived in favor of [`miragejs`](https://miragejs.com/). The documentation for `miragejs` is not only prodigious for its extensive API docs but also for its tutorials and guides. Furthermore, it receives regular bug-fixes and maintenance releases in thanks to its extremely large and active community.

Previous PR: [@folio-org/stripes-smart-components #894](https://github.com/folio-org/stripes-smart-components/pull/894)
_Linking the previous pull request so that subsequent ones can be found tagged below._

<details>
<summary>Overview of progress at the time of the creation of this PR (in chronological order):</summary>
<br>
✅ <code>ui-checkin</code>
<br>
✅ <code>ui-checkout</code>
<br>
☑️ <code>ui-circulation</code>
<br>
✅ <code>ui-calendar</code>
<br>
✅ <code>ui-agreements</code>
<br>
☑️ <code>ui-app-template</code>
<br>
✅ <code>ui-inventory</code>
<br>
✅ <code>ui-data-export</code>
<br>
✅ <code>ui-finc-config</code>
<br>
✅ <code>ui-licenses</code>
<br>
✅ <code>ui-erm-comparisons</code>
<br>
✅ <code>ui-myprofile</code>
<br>
✅ <code>ui-local-kb-admin</code>
<br>
☑️ <code>ui-marccat</code>
<br>
☑️ <code>ui-finc-select</code>
<br>
✅ <code>ui-erm-usage</code>
<br>
☑️ <code>ui-notes</code>
<br>
☑️ <code>ui-oai-pmh</code>
<br>
✅ <code>ui-plugin-find-user</code>
<br>
✅ <code>ui-plugin-find-eresource</code>
<br>
☑️ <code>ui-plugin-find-package-title</code>
<br>
☑️ <code>ui-plugin-find-import-profile</code>
<br>
✅ <code>ui-search</code>
<br>
✅ <code>ui-tenant-settings</code>
<br>
✅ <code>ui-requests</code>
<br>
☑️ <code>ui-eholdings</code>
<br>
☑️ <code>ui-data-import</code>
<br>
☑️ <code>stripes-smart-components</code>
<br>
☑️ <code>stripes-data-transfer-components</code>
</details>

## Approach
- [x] Cloned, installed, and then ran tests to double check that the tests are passing before refactoring.
- [x] Removed `@bigtest/mirage` and mirage config files per @VictorSoroka1's request.
- [x] ESlint & ran tests after the refactor to confirm they all pass.
- [x] Updated CHANGELOG.

### CODE REVIEW

**Comment by minkimcello** [comment]
@zburke Can't tag anyone to review. Could you do the honors?

**Comment by zburke** [comment]
Nor I, @minkimcello. Recent contributors are @VictorSoroka1, @YevheniiMaltsev, @Vladyslav-Velytskyi, @Yuliia-Dovhal. Can one of you merge this if you are happy with it? Thank you!

**Comment by VictorSoroka1** [comment]
We have a [story](STDTC-14) for this purpose. Please use it for the title, commit messages. According to it all mirage stuff should be removed. Not only the dependency but also the mirage configs.

**Comment by minkimcello** [comment]
@VictorSoroka1 done

**Comment by VictorSoroka1** [comment]
I that the result of the PR does not reflect the story's purpose @minkimcello.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## 2.1.0 (IN PROGRESS)
 * Update list formatters and list properties generators to fix rows accessibility. UIDEXP-117.
 * Update export and imports to make them consistent using named approach. STDTC-3.
+* Replace `@bigtest/mirage` with `miragejs`. STDTC-14
```
Comment (VictorSoroka1) [inline CHANGELOG.md:6]:
```suggestion
* Replace `@bigtest/mirage` with `miragejs`. STDTC-14.
```

**Comment by minkimcello** [comment]
> I see that the result of the PR does not reflect the story's purpose

You'll need to elaborate

File: package.json [hunk 2]
```diff
@@ -19,35 +19,36 @@
...
     "@folio/stripes-core": "^5.0.0",
     "@folio/stripes-smart-components": "^4.0.0",
-    "@folio/stripes": "^4.0.0",
     "babel-eslint": "^10.0.3",
     "chai": "^4.2.0",
-    "mocha": "^5.2.0",
+    "core-js": "^3.6.1",
     "eslint": "^6.2.1",
     "eslint-plugin-babel": "^5.3.0",
+    "inflected": "^2.0.4",
```
Comment (VictorSoroka1) [inline package.json:34]: This one is not needed.

**Comment by minkimcello** [comment]
@VictorSoroka1 You'll need to merge it as I don't have write access

**Comment by YevheniiMaltsev** [comment]
@minkimcello thank you for needed updates

**Comment by VictorSoroka1** [comment]
Thanks @minkimcello, done.

**Pull request merged**